### ISSUE TITLE
Cannot Bypass Addresses When Using Proxy

### ISSUE THREAD

**Issue by robbytarigan** [SeleniumHQ/selenium#5645]
## Meta -
OS:  
Windows 10
Selenium Version:  
3.11.0
Browser:  
Chrome

Browser Version:  
Version 65.0.3325.162

## Expected Behavior -
Can bypass addresses when using proxy

## Actual Behavior -
Receives `InvalidOperationException`
unknown error: cannot parse capability: proxyfrom unknown error: 'noProxy' must be a string  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4...

## Steps to reproduce -
Using Selenium.Webdriver Nuget https://www.nuget.org/packages/Selenium.WebDriver/, the code below will throws exception

```csharp
OpenQA.Selenium
OpenQA.Selenium.Chrome

void Main()
{
	string ADVERT_PROXY_URL = "someproxy:9111";	
	var options = new ChromeOptions();	

	var optionsProxy = new Proxy { HttpProxy = ADVERT_PROXY_URL };
	optionsProxy.AddBypassAddresses("*.mydev.ds");	
	options.Proxy = optionsProxy;
	
	var driver = new ChromeDriver(options); // Throws exception here
}
```

**Comment by jimevans** [comment]
Fixed in f454a5c.

**Comment by neilkennedy** [comment]
Hi @jimevans, do you know when this will make it into a release?
